build/build-image/Dockerfile

```diff
@@ -28,5 +28,9 @@ RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.
 
 RUN go get -u github.com/golang/lint/golint
 
+RUN git clone https://github.com/duglin/vlinker.git /vlinker
+
+ENV PATH=$PATH:/vlinker/bin
+
```

Comment:
i.e. I think split is fine, if not preferable. These layers are purely additive, so any space concerns are minimal compared to any type of 'download and delete' path.